scripts/init.sh

```diff
@@ -21,6 +21,8 @@ $ALLORAD_BIN genesis add-genesis-account allo1q4fa4tqzng2lshfhjaklx90hzfnfennxt0
 $ALLORAD_BIN genesis add-genesis-account allo1zy5akp9grwfp3x6rqd40x0g4agpzjaskxr9lnn 10000000allo
 $ALLORAD_BIN genesis add-genesis-account allo1ywhj2svg67mn7ylr9mu5kz9f668z2xejnp9w9y 10000000allo
 $ALLORAD_BIN genesis add-genesis-account allo1r7hqeqdmf6jg9v9px0gh5l6n7tlr0tlxt86plc 10000000allo
+# Ecosystem account gets 36.75% of total supply
+$ALLORAD_BIN genesis add-genesis-account allo12uxa8rw9hte3z2nuswjzpmfen289n30uag6agp 367500000allo
```

Comment:
Might be more of a business question, but the optics around this are kind bad IMO. These should be put into a vesting account that unlocks community funds over time. Tossing 1/3 into a org controlled wallet will bring a lot of FUD. Maybe this is just pre production?

Reply:
a) this is a testing environment
b) this address is not human controlled, it's the address of the module account which is exactly the vesting account you described

Reply:
thanks for the clarification. Is this key constant, shouldn't this be in the modules genesis initializer?

Reply:
no i think it belongs to the process making the genesis.json

also this specific line of code deleted after some discussion with diederik

Reply:
> no i think it belongs to the process making the genesis.json
> 
> also this specific line of code deleted after some discussion with diederik

Hmm, right I thought there is a genesis initializer for each module?